### PR TITLE
🌐 Add Spanish translation for `advanced/additional-status-codes.md`

### PR DESCRIPTION
**Pull request by jfunez**

### CODE REVIEW

File: docs/es/docs/advanced/additional-status-codes.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Códigos de Estado Adicionales
+
+Por defecto, **FastAPI** retornará las respuestas utilizando una `JSONResponse`, poniendo el contenido que retornas en tu *operación de camino* dentro de esa `JSONResponse`.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/additional-status-codes.md:3]:
```suggestion
Por defecto, **FastAPI** retornará las respuestas utilizando una `JSONResponse`, poniendo el contenido que retornas en tu *path operation* dentro de esa `JSONResponse`.
```

File: docs/es/docs/advanced/additional-status-codes.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Utilizará el código de estado por defecto, o el que sea asignado en tu *operación de camino*.
+
+## Códigos de estado adicionales
+
+Si tu quieres retornar códigos de estado adicionales además del principal, puedes hacerlo retornando directamente una `Response`, como una `JSONResponse`, y asignar directamente el código de estado adicional.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/additional-status-codes.md:9]:
```suggestion
Si quieres retornar códigos de estado adicionales además del principal, puedes hacerlo retornando directamente una `Response`, como una `JSONResponse`, y asignar directamente el código de estado adicional.
```

File: docs/es/docs/advanced/additional-status-codes.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Por ejemplo, digamos que tu quieres tener una *operación de camino* que permita actualizar ítems, y retornar códigos de estado HTTP 200 "OK" cuando sea satisfactorio.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/additional-status-codes.md:11]:
```suggestion
Por ejemplo, digamos que quieres tener una *path operation* que permita actualizar ítems, y retornar códigos de estado HTTP 200 "OK" cuando sea satisfactorio.
```

File: docs/es/docs/advanced/additional-status-codes.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+```Python hl_lines="2  19"
+{!../../../docs_src/additional_status_codes/tutorial001.py!}
+```
+
+!!! warning
+    Cuando tu retornas directamente una `Response`, como en los ejemplos anteriores, será retornado directamente.
+
+    No será serializado con el modelo, etc.
+
+    Asegurate que la respuesta tenga los datos que tu quieras, y que los valores sean JSON válidos (si estás usando `JSONResponse`).
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/additional-status-codes.md:26]:
```suggestion
    Asegurate que la respuesta tenga los datos que quieras, y que los valores sean JSON válidos (si estás usando `JSONResponse`).
```

File: docs/es/docs/advanced/additional-status-codes.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+!!! note "Detalles Técnicos"
+    También puedes utilizar `from starlette.responses import JSONResponse`.
+
+    **FastAPI** provee las mismas `starlette.responses` que `fastapi.responses` simplemente como una convención para ti, el desarrollador. Pero la mayoría de las respuestas disponibles vienen directamente de Starlette. Lo mismo con `status`.
+
+## OpenAPI y documentación de API
+
+Si tu quieres retornar adicionalmente códigos de estado y respuestas directamente, no estarán incluidas en el esquema OpenAPI (documentación de API), porque FastAPI no tiene una manera de conocer de antemano lo que tu vas a retornar.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/additional-status-codes.md:35]:
```suggestion
Si quieres retornar adicionalmente códigos de estado y respuestas directamente, no estarán incluidas en el esquema OpenAPI (documentación de API), porque FastAPI no tiene una manera de conocer de antemano lo que vas a retornar.
```

File: docs/es/docs/advanced/additional-status-codes.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+Pero tu puedes documentar eso en tu código, usando [Respuestas Adicionales](additional-responses.md){.internal-link target=_blank}.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/additional-status-codes.md:37]:
```suggestion
Pero puedes documentar eso en tu código, usando [Respuestas Adicionales](additional-responses.md){.internal-link target=_blank}.
```

**Review by mariacamilagl — CHANGES_REQUESTED** [review]
Thanks for the translation @jfunez! 🚀 

Minor changes like the last ones.

File: docs/es/docs/advanced/additional-status-codes.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+# Códigos de Estado Adicionales
+
+Por defecto, **FastAPI** retornará las respuestas utilizando una `JSONResponse`, poniendo el contenido que retornas en tu *operación de camino* dentro de esa `JSONResponse`.
+
+Utilizará el código de estado por defecto, o el que sea asignado en tu *operación de camino*.
```
Comment (pedabraham) [inline docs/es/docs/advanced/additional-status-codes.md:5]:
```suggestion
Utilizará el código de estado por defecto, o el que sea asignado en tu *path operation*.
```

**Review by pedabraham — COMMENTED** [review]
Hice una pequeña corrección.

**Review by mariacamilagl — APPROVED** [review]
Thanks for the effort @jfunez! I implemented some changes and fixed a couple of grammar mistakes. 

This is ready now! 🎉

**Comment by tiangolo** [comment]
Thanks for the contribution @jfunez! :cake: 

And thanks a lot for the reviews and fixes @mariacamilagl and @pedabraham! :bow: :cookie:

**Pull request merged**